resources/dicts/events/death/death_reactions/general/general_dislike.json

```diff
@@ -6,7 +6,11 @@
       "r_c wanders past m_c's vigil. {PRONOUN/r_c/subject/CAP} won't mourn m_c's passing.",
       "Frankly, they're all better off without m_c, and r_c doesn't know why everyone else is so grief stricken.",
       "As much as {PRONOUN/r_c/subject} hated m_c, the sight of {PRONOUN/m_c/poss} body still fills r_c with dread.",
-      "Yes, {PRONOUN/r_c/subject} hated m_c, but m_c was {PRONOUN/r_c/poss} Clanmate first and foremost, and r_c bows {PRONOUN/r_c/poss} head at the vigil respectfully."
+      "Yes, {PRONOUN/r_c/subject} hated m_c, but m_c was {PRONOUN/r_c/poss} Clanmate first and foremost, and r_c bows {PRONOUN/r_c/poss} head at the vigil respectfully.",
+      "With m_c sadly gone from this world, r_c didn't take much to openly mock the decease cat's flaws and mistakes for the Clan. With them gone, the Clan can run perfectly now.",
```

Comment:
rather than "r_c didn't take much", you should try "it didn't take much for r_c".  the last "them" also needs to be pronoun tagged